### PR TITLE
Use sensirion-i2c for crc8 and basic I2C functions

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -193,6 +192,17 @@ pub enum Error<E> {
     NotInitialized,
 }
 
+impl<E, I2cWrite: Write<Error = E>, I2cRead: Read<Error = E>> From<i2c::Error<I2cWrite, I2cRead>> for Error<E>
```
[the remark this answers is not in the document]
Comment (rnestler) [reply src/lib.rs]: Fixed.

**Comment by rnestler** [comment]
@dbrgn Ping :slightly_smiling_face:

**Comment by dbrgn** [comment]
I wanted to wait for https://github.com/Sensirion/sensirion-i2c-rs/issues/10, since it might include API changes... But if we use a panic based approach and not an error, then nothing will change :slightly_smiling_face:

**Review by dbrgn — APPROVED** [review]
Thanks!

**Comment by dbrgn** [comment]
Released as 0.3.1.

**Pull request merged**